projects/jenkins.io/package.yml

```diff
@@ -46,14 +46,17 @@ test:
     curl.se: '*'
   script:
     - run: |
-        for i in $(seq 1 10); do
-          FREE_PORT=$((RANDOM % 10000 + 1024))
-          if ! nc -z localhost $FREE_PORT; then
-            echo $FREE_PORT
-          fi
+        FREE_PORT=$((RANDOM % 10000 + 1024))
```

Comment:
there's an `npx` utility that can do this. I remember not what it is called but we use it in the sd-webui endpoint script.

Reply:
`get-port`. not sd-webui, but others. gotta bring in the node ecosystem then, of course.